src/components/React/ControlsPanel.js

```diff
@@ -1,74 +1,82 @@
+            Options
+          </Button>
+        </Col>
 
-  volumeSelectChanged = (selected) => {
-    this.setState({
-      currentVolume: selected.value,
-      currentData:selected
-    });
-    
-    if(selected.value != null) {
-      var volumeProperties = selected.value.split(":");
-      this.props.myChangeVolumeAction(volumeProperties[0],volumeProperties[1]);
-    } else {
-      this.props.myChangeVolumeAction("", "");
-    }
-  };
+        <Col>
+          <ToggleButtonGroup 
+            type="radio" 
+            name="variable"
+            value={toggles.measurement}
+            onChange={val => dispatch({
+              type: "TOGGLE_MEASUREMENT",
+              payload: val,
+            })}
+          >
+            <ToggleButton value={0}>Salinity</ToggleButton>
+            <ToggleButton value={1}>Temperature</ToggleButton>
+          </ToggleButtonGroup>
+        </Col>
 
-  render () {
-    return (
-      <div className="mx-1">
-        <Flexbox justifyContent="center" className="my-3 mx-3">
-          <Button 
-            icon="pi pi-arrow-right" 
-            label="Controls" 
-            onClick={(e) => this.setState({sideBarVisible: true})}
-          />
-          <SelectButton 
-            value={this.state.currentVolume}
-            options={options} 
-            onChange={this.volumeSelectChanged} 
-          /> 
-        </Flexbox>
+        <Col className="text-center">
+          <ToggleButtonGroup 
+            type="radio" 
+            name="season" 
+            value={toggles.season} 
+            onChange={val => dispatch({
+              type: "TOGGLE_SEASON",
+              payload: val,
+            })}
+          >
+            <ToggleButton value={0}>Summer</ToggleButton>
+            <ToggleButton value={1}>Winter</ToggleButton>
+          </ToggleButtonGroup>
+        </Col>
 
-        <Sidebar
-          modal={false} 
-          position="bottom"
-          visible={this.state.sideBarVisible} 
-          onHide={(e) => this.setState({sideBarVisible:false})}
-          style={{width:'20em', height:'45em'}}
-        >
-          <Controls/>
-        </Sidebar>
-      </div>
-    );
-  }
-})
\ No newline at end of file
+        <Col className="text-center">
+          <ToggleButtonGroup 
+            type="radio" 
+            name="tide"
+            value={toggles.tide}
+            onChange={val => dispatch({
+              type: "TOGGLE_TIDE",
+              payload: val,
+            })}
+          >
+            <ToggleButton value={0}>Low Tide</ToggleButton>
+            <ToggleButton value={1}>High Tide</ToggleButton>
+          </ToggleButtonGroup>
+        </Col>
+      </Row>
+
+      <Sidebar
+        modal={false}
+        position="bottom"
+        visible={sidebarVisible} 
+        onHide={(e) => setSidebarVisible(false)}
+        style={{width:'20em', height:'45em'}}
+      >
+        <Controls/>
+      </Sidebar>
```

Comment:
I'm interested in the organization of components here. Are the toggles not part of the sidebar?

Reply:
No the toggles are on the top of the web page (laid out horizontally). Definitely a design decision to make down the road

Reply:
Agreed - should probably think about where to best put the sidebar in the code since it's not really in this component.